### PR TITLE
Add glossary & onboarding section

### PR DESCRIPTION
**Pull request by zanetworker**
This PR adds:

- an onboarding section for new Joiners of the working group.
- a separate glossary page

### CODE REVIEW

File: website/content/wgs/cnaiwg/onboarding/index.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+title: CNCF Cloud Native AI Working Onboarding
```
Comment (raravena80) [inline website/content/wgs/cnaiwg/onboarding/index.md:2]: Could we just title this 'Onboarding'

The left navigation on the CN AI WG page has the almost the same initial text for all links `CNCF Cloud Native AI Working..`

File: website/content/wgs/cnaiwg/glossary/index.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
+---
+title: CNCF Cloud Native AI Working Glossary
```
Comment (raravena80) [inline website/content/wgs/cnaiwg/glossary/index.md:2]: Can we just name this 'Glossary'?

File: website/content/wgs/cnaiwg/onboarding/index.md (hunk 1 above)
Comment (zanetworker) [reply website/content/wgs/cnaiwg/onboarding/index.md:2]: Sure, I was just trying to be consistent but thought the same thing.

File: website/content/wgs/cnaiwg/onboarding/index.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+Welcome to the CNAI working group. If you are confused, or want to contribute, here are some recommended steps to get you started:
+
+## Step 1: Read the White Paper
+
+Start by reading the [white paper](https://tag-runtime.cncf.io/whitepapers/cloudnativeai/) on Cloud-Native AI to gain a comprehensive understanding of problems/opportunities and the foundation work done by the working group.
+
+## Step 2: Check the Most Recent Posts on the WG Blog
+
+Review the [most recent posts](https://tag-runtime.cncf.io/blog/cnai-kubecon-eu-paris-2024-recap/) on the Cloud-Native AI blog for updates and announcements. For example:
```
Comment (VijayRod) [inline website/content/wgs/cnaiwg/onboarding/index.md:16]: Perhaps the link can be updated to https://tag-runtime.cncf.io/blog/

**Review by raravena80 — APPROVED** [review]
LGTM.  Thanks for adding this.

**Pull request merged**